files/ipset_sync

```diff
@@ -36,6 +36,7 @@ function get_content_file() {
 
 function construct_ipset_dump() {
   local id=$1
+  local setname=${2:-$id}
```

Comment:
I don't understand reasoning behind this second parameter. Can you please elaborate? Thanks.